etc/cmake/options.cmake

```diff
@@ -174,6 +174,7 @@ ot_option(OT_BACKBONE_ROUTER_DUA_NDPROXYING OPENTHREAD_CONFIG_BACKBONE_ROUTER_DU
 ot_option(OT_BACKBONE_ROUTER_MULTICAST_ROUTING OPENTHREAD_CONFIG_BACKBONE_ROUTER_MULTICAST_ROUTING_ENABLE "BBR MR")
 ot_option(OT_BLE_TCAT OPENTHREAD_CONFIG_BLE_TCAT_ENABLE "Ble based thread commissioning")
 ot_option(OT_BORDER_AGENT OPENTHREAD_CONFIG_BORDER_AGENT_ENABLE "border agent")
+ot_option(OT_BORDER_AGENT_EPSK OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE "border agent ephemeral PSK")
```

Comment:
A small question: when should we call it `ePSK` and when should we call it `ePSKC`?

Reply:
Either one is fine. The "c" in `PSKc` stands for commissioner/commissioning. This is to distinguish it from a "PSKd" used by joiner devices. I believe on a Border Agent, PSK will always refer to the "commissioner" PSK.

Reply:
Thanks for the explanation!

Reply:
Given the specs refer to it as `ePSKc`, is it better to keep consistent with the spec?

Reply:
Changed to `EPSKC` in new push.